### PR TITLE
Peer count in client status

### LINKED ISSUE #7422: Remove peers list from coda client status
It should still show the number of peers, but not display the full list of peer strings. The current setup makes coda client status much harder to read/interact with, and the full peers list is still available with `coda advanced get-peers`

### PR DESCRIPTION
**Pull request by psteckler**
In `client status`, print only the number of peers, not the actual peers.

Tested by running a local network.

Output:
```
Peers:                                         3
```

Closes #7422.

### CODE REVIEW

File: src/lib/daemon_rpcs/types.ml [hunk 1]
```diff
@@ -213,18 +213,7 @@ module Status = struct
...
-      in
-      map_entry
-        (sprintf "Peers (%d)" (List.length @@ FieldT.get field))
-        ~f:(fun peers ->
-          List.mapi peers ~f:(fun i peer ->
-              let rendered = "\t" ^ render peer in
-              if i = 0 then "\n" ^ rendered else rendered )
-          |> String.concat ~sep:"\n" )
-        field
+      Some ("Peers", sprintf "%d" @@ List.length (FieldT.get field))
```
Comment (mrmr1993) [inline src/lib/daemon_rpcs/types.ml:216]: Can you make it optional, toggled by a flag?

**Review by lk86 — APPROVED** [review]

**Comment by lk86** [comment]
Does this change the graphql output for clientStatus or whatever it's called there? I'm expecting not as the data structure is the same, but wanted to make sure.

File: src/lib/daemon_rpcs/types.ml (hunk 1 above)
Comment (psteckler) [reply src/lib/daemon_rpcs/types.ml]: will do

**Comment by psteckler** [comment]
> Does this change the graphql output for clientStatus or whatever it's called there?

This change affects only the CLI output.

File: src/lib/daemon_rpcs/types.ml (hunk 1 above)
Comment (psteckler) [reply src/lib/daemon_rpcs/types.ml]: seems too difficult to pass a flag here

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**